src/parser/jsoncons/json_parser.hpp.orig

```diff
@@ -0,0 +1,2157 @@
+// Copyright 2015 Daniel Parker
```

Comment:
I removed this file, and simplifyDriver is compiling with no problems. I think, the core json functionality that we are using, does not depend on this file.